### PR TITLE
C51-326: Activity Feed width fix

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of the PR the Activity List has been made responsive
When Width is more than 1400px
 * Width of left panel should be 50%
 * Width of right panel should be 30%

When Width is less than or equal to `1400px`
 * Width of left panel should be 60%
 * Width of right panel should be 40%

## Before
before

## After
after

## Technical Details
1. `civicase__activity-feed__list-container__left` and `civicase__activity-feed__list-container__right` classes has been added to apply the styles.
2. `civicase__activity-feed__placeholder` is added to apply `50%` width to the placeholders.

### CODE REVIEW

File: scss/components/_activity-panel.scss [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
   margin-top: 37px;
-  width: 484px;
 
   .panel {
     overflow: auto;
-    width: 484px;
 
     &.affix {
       position: fixed !important;
+      width: calc(30% - 12px);
```
Comment (swastikpareek) [inline scss/components/_activity-panel.scss]: Can we drop a little comment why we are subtracting `12px` specifically?

File: scss/components/_activity-feed.scss [hunk 2]
```diff
@@ -102,11 +103,11 @@
     margin-bottom: 5px;
     margin-left: auto;
     margin-right: auto;
-    width: 668px;
+    width: calc(100% - 50px);
```
Comment (swastikpareek) [inline scss/components/_activity-feed.scss]: Can we drop a little comment why we are subtracting `50px` specifically?

File: scss/components/_activity-feed.scss [hunk 3]
```diff
@@ -44,7 +43,8 @@
 
 .civicase__activity-feed__activity-container {
   display: inline-block;
-  width: 765px;
+  margin-left: 5px;
+  width: calc(100% - 50px);
```
Comment (swastikpareek) [inline scss/components/_activity-feed.scss]: Can we drop a little comment why we are subtracting `50px` specifically?

File: scss/components/_activity-panel.scss (hunk 1 above)
Comment (deb1990) [reply scss/components/_activity-panel.scss]: @swastikpareek Honestly this is a magic number. I have tried to come up with a explanation but could not find one after spending 10-15 min. The panel has 30% width, but when its affixed, we get a slight jitter unless we specify `-12px`.

**Pull request merged**